src/Codeception/Module/Silex.php

```diff
@@ -23,6 +24,7 @@
  * ## Config
  *
  * * app: **required** - path to Silex bootstrap file.
+ * * em_service: 'db.orm.em' - use the stated EntityManager to pair with Doctrine Module.
  *
  * ### Bootstrap File
```

Comment:
Could you provide here example configuration for Silex + Doctrine2? This would be good for newbies to understand how to use them together.